zulipterminal/model.py

```diff
@@ -654,6 +654,16 @@ def _handle_typing_event(self, event: Event) -> None:
                 else:
```

Comment:
Typo: I think the commit description should be 

> currently **no** check in place

Reply:
No space before bracket currently.